eclipse/sprotty
# Implemented client for running ELK layout in a separate process

This PR adds two classes `SocketElkServer` and `StdioElkServer` to `sprotty-elk` that enable to run the ELK layout engine in a separate Java process. The Java application is implemented in a new MIT-licensed project [elk-server](https://github.com/TypeFox/elk-server).

## How To Test

1. Download [elk-server](https://github.com/TypeFox/elk-server/releases/tag/v0.1.0)
2. Run the application with the `--socket` argument:
   ```
    elk-server-0.1.0/bin/elk-server --socket
   ```
3. Build and run the examples server (`yarn start`)
4. Open the new "random graph" example in the "With Server" section on http://localhost:8080/

The result should look similar to the previous in-browser "random graph" example.
